src/raoblackwellized.jl

```diff
@@ -0,0 +1,75 @@
+abstract type RaoBlackwellizedParticleBelief end
+
+struct RaoBlackwellizedParticleFilter <: Updater
+    particle_filter::BasicParticleFilter
+    analytical_filter::AbstractFilter # https://github.com/sisl/GaussianFilters.jl/blob/master/src/kf_classes.jl#L7
+end
```

Comment:
```suggestion
struct RaoBlackwellizedParticleFilter{PF, AF} <: Updater
    particle_filter::PF
    analytical_filter::AF
end
```
abstract field types can negatively effect performance. We should have a docstring explaining what is expected of `particle_filter` and `analytical_filter`.